config/database.yml

```diff
@@ -1,25 +1,21 @@
-# SQLite. Versions 3.8.0 and up are supported.
-#   gem install sqlite3
-#
-#   Ensure the SQLite 3 gem is defined in your Gemfile
-#   gem 'sqlite3'
-#
 default: &default
-  adapter: sqlite3
+  adapter: postgresql
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
   timeout: 5000
 
 development:
   <<: *default
-  database: db/development.sqlite3
+  database: devrailsblog
 
 # Warning: The database defined as "test" will be erased and
 # re-generated from your development database when you run "rake".
 # Do not set this db to the same as development or production.
 test:
   <<: *default
-  database: db/test.sqlite3
+  database: testrailsblog
 
 production:
   <<: *default
-  database: db/production.sqlite3
+  database: dbbgov5vn124re
```

Comment:
Is it necessary to define both `database` and `url`?

Reply:
I reviewed the Heroku Config vars and is a more secure set only URL because the database name in the file can be dangerous.

Reply:
And because if you have the `url` you don't need to set the name of the database.